### DISCUSSION TITLE
Getter does not work in Setup Store

### DISCUSSION THREAD

**Discussion by yxyx-github** [vuejs/pinia#1496] [Help and Questions] [answered]
### Reproduction

https://github.com/yxyx-github/pinia-getter-issue

### Steps to reproduce the bug

Create a store with Pinia and define a getter which returns any value. Use the Setup Store.

In the reproduction repository the store is located at `src/stores/demo-store.ts`. You can comment in/out the two versions: Option Store and Setup Store.
I try to access the getter in the `App.vue`.

### Expected behavior

Regardless of whether I use the Option Store or the Setup Store, I should have access to my getter in any Vue-Component.

### Actual behavior

When I use the Option Store, my getter works fine. On the other hand, when I use the Setup Store, my getter returns undefined. I do not get any other error messages.
I can't find an example of a getter declaration within a setup store in the documentation to verify that I have declared my getter correctly.

**Comment by posva** [comment, marked as answer]
You have to return the value:

```diff
export const useStore = defineStore('default', () => {
    const sampleArg = ref('hello world')

    const demoGetter = computed(() => sampleArg)
+	return { sampleArg, demoGetter }
})
```

**Reply by yxyx-github** [reply]
Thank you so much! I am so stupid...
I use `<script setup>` in Vue too often where I don't need to return anything.
